README.md

```diff
@@ -295,17 +295,16 @@ Outputs:
 Generate a customizable progress bar and display it
 
 ```typescript
-const options = {
-                    format: 'Progress | {bar} | {percentage}%',
-                    barCompleteChar: '\u2588',
-                    barIncompleteChar: '\u2591'
-                }
-const bar = new cli.progress('SingleBar', options);
+const bar = new cli.progress({
+                 format: 'Progress | {bar} | {percentage}%',
+                 barCompleteChar: '=',
+                 barIncompleteChar: '-'
+             });
 ```
 
 Outputs:
```

Comment:
The above code doesn't produce this output. You should provide two examples: 1) the simplest way to use it `cli.progress()` and 2) a more detailed example like you have now.